src/test/java/decodes/tsdb/groupedit/GroupEvalTsidsDialogScaffold.java

```diff
@@ -0,0 +1,42 @@
+package decodes.tsdb.groupedit;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.swing.SwingUtilities;
+
+import decodes.cwms.CwmsTimeSeriesDb;
+import decodes.cwms.CwmsTsId;
+import decodes.db.DataType;
+import decodes.sql.DbKey;
+import decodes.tsdb.TimeSeriesIdentifier;
+
+final class GroupEvalTsidsDialogScaffold
+{
+	public static void main(String[] args)
+	{
+		SwingUtilities.invokeLater(new Runnable()
+		{
+			@Override
+			public void run()
+			{
+
+				List<String> timeSeriesIdentifiers = Arrays.asList("Barren.Precip.Inst.1Day.0.lrldlb-raw",
```

Comment:
@agilmore2 is there a common set of USBR timeseries id we can include in this test?

Reply:
I believe TS_IDs are just a primary key in HDB. I don't think TS_ID by itself will communicate much. There's not a common ts_id to any installations outside the Colorado.

Reply:
Surely there is a set of information that could be used to verify we aren't breaking anything for HDB users?
I'm pretty sure no. It looks like Adam just shifted the GUI elements around but otherwise kept the same information..

Reply:
Some HDB TSIDs:
919.49.month.R_.-1
BTBLESCO.19.hour.M_.10
BIGLOVCO.1049.instant.R_.-1
PUERESCO.17.day.R_.-1

Thanks for the question!

Reply:
Thanks. We had to connect to a full instance of CWMS/CCP to test the CWMS ones. For something at this level though I'm hoping we can come up with a simple set of mocks to let us all test these things easier. But that will be a later project.